pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -214,33 +215,47 @@ func (s *AsyncAgentStore) GetUsers(limit int) []api.User {
 type PollerWorker struct {
 	// AgentCommands is a list of commands executed on the guestAgent
 	AgentCommands []AgentCommand
-	// CallTick is how often to call this set of commands
+
+	// CallTick is how often to call the this
 	CallTick time.Duration
 }
 
 type agentCommandsExecutor func(commands []AgentCommand)
 
-// Poll is the call to the guestagent.
+type apiOperationsExecutor func()
+
+// Poll is the call to the guest agent using qemu agent commands.
 func (p *PollerWorker) Poll(execAgentCommands agentCommandsExecutor, closeChan chan struct{}, initialInterval time.Duration) {
-	log.Log.Infof("Polling command: %v", p.AgentCommands)
+	genericPoller(func() { execAgentCommands(p.AgentCommands) }, closeChan, initialInterval, p.CallTick, "Agent poller started with commands")
+}
+
+// pollApiOperation is the call to the guest agent using libvirt API.
+func pollApiOperation(execApiOperation apiOperationsExecutor, closeChan chan struct{}, initialInterval, callTick time.Duration) {
+	genericPoller(execApiOperation, closeChan, initialInterval, callTick, "Polling API operation")
+}
```

Comment:
This function can be inlined. It only passes parameters to the other function.

Reply:
Done.